package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@hmcts/postcodeinfo-client",
+  "version": "0.0.1",
+  "description": "MoJ postcode lookup client",
+  "license": "MIT",
+  "repository": "https://github.com/hmcts/postcodeinfo-client-node",
+  "files": [
+    "lib"
+  ],
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "clean": "rimraf lib && rimraf coverage",
```

Comment:
Why not just use `rm`?

Reply:
the generator added it, I guess crossplatform? haven't really looked at it